src/hooks/useApi.tsx

```diff
@@ -0,0 +1,33 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+import axios, { AxiosResponse } from 'axios';
+import { useQuery } from 'react-query';
+
+import { toastNotify } from '../utils/ToastNotify';
+
+export function useRequest<T, G>(
+    callback: (config: G) => Promise<AxiosResponse<T, G>>,
+    args: G,
+    tag: string,
+    toastToggle: boolean
```

Comment:
that should be optional with default value of false

Also I think that the arguments could be optional too, don't we have any request which doesn't require any request body? (some GET or smth?)